release/docker/init_script.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Read the user given arguments
+ARGS=$@
+# Read the configuration @ /etc/memgraph/memgraph.conf
+FLAGS="$(perl -ne 'print "$1=$2"  while /^(-{2}\b[a-z][a-z\d]{2,}\b(?:-[a-z\d]+)*)=((?:[a-zA-Z][a-zA-Z0-9_]*$)|(?:(?:(?:[\/]*\/)*)(?:.*))|(?:[0-9]*))/g' /etc/memgraph/memgraph.conf | sed '/^\s*$/d')"
```

Comment:
what

Reply:
The most useless two hours spent in Memgraph. The goal was extracting a port from the gflags  but I have extracted all gflags arguments from config. Reason being checking if Memgraph is active during init script. Ended up with this disaster, and yeah, now I have no idea why I needed ALL arguments, at the point of making it was pretty clear for me...